charts/utils/templates/deployments.yaml

```diff
@@ -0,0 +1,59 @@
+{{ if .Values.deployments.entities }}
+{{ $prefix := printf "%s-%s" $.Values.client $.Values.environment }}
+{{- range $deployment := .Values.deployments.entities }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $prefix }}-{{ $deployment.name }}-client
+  {{- if $.Values.namespace }}
+  namespace: {{ $.Values.namespace }}
+  {{- end }}
+  labels:
+    app: {{ $prefix }}-{{ $deployment.name }}-client
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ $prefix }}-{{ $deployment.name }}-client
+  template:
+    metadata:
+      labels:
+        app: {{ $prefix }}-{{ $deployment.name }}-client
+    spec:
+      nodeSelector:
+        databases: 'true'
+      securityContext:
+        runAsUser: 999
```

Comment:
that should be the value of the chart, not hardcoded. You can use the default value of 999 or 1000 if it's not provided.